packages/jest-expo/babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    // Only use this for the `tests/` folder.
```

Comment:
```suggestion
    // Only use this when running tests
```
(Jest in any folder will look for `env -> test`)